1.0.19 build failure on test

I got jcodings 1.0.19 build failure as below (built on Debian unstable).

```
[INFO] ------------------------------------------------------------------------
[INFO] Building JCodings 1.0.19
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ jcodings ---
[WARNING] Using platform encoding (ANSI_X3.4-1968 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] Copying 559 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.2:compile (default-compile) @ jcodings ---
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding ANSI_X3.4-1968, i.e. build is platform dependent!
[INFO] Compiling 128 source files to /build/jcodings-1.0.19/target/classes
[INFO] /build/jcodings-1.0.19/src/org/jcodings/util/Hash.java: Some input files use unchecked or unsafe operations.
[INFO] /build/jcodings-1.0.19/src/org/jcodings/util/Hash.java: Recompile with -Xlint:unchecked for details.
[INFO]
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ jcodings ---
[WARNING] Using platform encoding (ANSI_X3.4-1968 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /build/jcodings-1.0.19/src/test/resources
[INFO]
[INFO] --- maven-compiler-plugin:3.2:testCompile (default-testCompile) @ jcodings ---
[INFO] Changes detected - recompiling the module!
[WARNING] File encoding has not been set, using platform encoding ANSI_X3.4-1968, i.e. build is platform dependent!
[INFO] Compiling 9 source files to /build/jcodings-1.0.19/target/test-classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[10,30] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[10,31] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[10,32] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[10,33] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,23] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,24] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,22] unclosed character literal
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,24] illegal character: '\ufffd'
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,25] unclosed character literal
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,40] ';' expected
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,76] ';' expected
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestEConv.java:[86,35] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestEConv.java:[86,36] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestEConv.java:[86,37] unmappable character for encoding ASCII
[INFO] 14 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.298 s
[INFO] Finished at: 2017-04-15T02:05:13+00:00
[INFO] Final Memory: 13M/303M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.2:testCompile (default-testCompile) on project jcodings: Compilation failure: Compilation failure:
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[10,30] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[10,31] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[10,32] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[10,33] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,23] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,24] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,22] unclosed character literal
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,24] illegal character: '\ufffd'
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,25] unclosed character literal
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,40] ';' expected
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestUnicode.java:[15,76] ';' expected
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestEConv.java:[86,35] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestEConv.java:[86,36] unmappable character for encoding ASCII
[ERROR] /build/jcodings-1.0.19/test/org/jcodings/specific/TestEConv.java:[86,37] unmappable character for encoding ASCII
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/pom.xml
+++ b/pom.xml
@@ -94,6 +94,7 @@
         <configuration>
           <source>1.5</source>
           <target>1.5</target>
+          <encoding>UTF-8</encoding>
         </configuration>
       </plugin>
       <plugin>

